packages/xod-patch-search/test/fixtures/indexData.json

```diff
@@ -0,0 +1,107 @@
+  },
+  {
+    "path": "xod/common-hardware/servo",
+    "keywords": ["servo"],
+    "lib": "xod/common-hardware",
+    "description": "Drives a standard hobby servo",
+    "fullDescription": "#Servo \n\nHas two inputs: \n\n*PORT (number)*\nBoard port number the servo is connected to.\n\n*VAL (number)*\nDesired servo angle or value in unit range [0.0, 1.0]. For standard servo 0.0 would be mapped to 0° and 1.0 would be 180°."
+  },
+  {
+    "path": "xod/common-hardware/h-bridge-dc-motor",
+    "keywords": ["h", "bridge", "dc", "motor"],
+    "lib": "xod/common-hardware",
+    "description": "Drives a DC motor through H-Bridge IC such as L293 or L298.",
+    "fullDescription": "#Servo \n\nHas three inputs: \n\n*DIR (number)*\nBoard port to which the direction pin of the H-Bridge is connected.\n\n*PWM (number)*\nBoard port to which the PWM (aka Enable) pin of the H-Bridge is connected. The port should be capable of producting PWM signal.\n\n*SPD (number)*\nDesired speed of the motor in [-1.0, 1.0] range. Negative values denote clockwise rotation, zero denotes full stop, positive values denote counterclockwise direction."
+  },
+  {
+    "path": "xod/common-hardware/gp2y0a41-range-meter",
+    "keywords": ["gp2y0a41", "range", "meter"],
+    "lib": "xod/common-hardware",
+    "description": "Reads Sharp infrared range meter GP2Y0A41SK0F (the one with 4…30 cm range).",
+    "fullDescription": "#gp2y0a41-range-meter \n\nMeasured distance in meters. Trustworhy only for distances in [0.04, 0.30] meters range."
+  },
+  {
+    "path": "xod/common-hardware/dht11-thermometer",
+    "keywords": ["dht11", "thermometer"],
+    "lib": "xod/common-hardware",
+    "description": "Reads temperature and humidity from a DHT11 sensor.",
+    "fullDescription": "#dht11-thermometer \n\nReturns last read temperature in °C and relative humidity in ragne [0.0, 1.0]."
+  },
+  {
+    "path": "xod/common-hardware/thermometer-tmp36",
+    "keywords": ["thermometer", "tmp36"],
+    "lib": "xod/common-hardware",
+    "description": "Drives a digital thermometer based on TMP36 IC",
+    "fullDescription": "#thermometer-tmp36 \n\nReturns last read temperature in °C."
+  },
+  {
+    "path": "xod/utils/deg-to-rad",
+    "keywords": ["deg", "to", "rad"],
+    "lib": "xod/utils",
+    "description": "Converts ° (degrees) to radians",
+    "fullDescription": ""
+  },
+  {
+    "path": "xod/utils/rad-to-deg",
+    "keywords": ["rad", "deg"],
+    "lib": "xod/utils",
+    "description": "Converts radians to ° (degrees)",
+    "fullDescription": ""
+  },
+  {
+    "path": "xod/utils/c-to-f",
+    "keywords": ["c", "f"],
+    "lib": "xod/utils",
+    "description": "Maps temperature from °C to °F",
+    "fullDescription": ""
+  },
+  {
+    "path": "xod/utils/m-to-cm",
+    "keywords": ["m", "cm"],
+    "lib": "xod/utils",
+    "description": "Maps meters to centimeters",
+    "fullDescription": ""
+  }
+]
```

Comment:
This fixture will be hard to maintain. It’s better to generate the index just in time from the real data in `./workspace`